docs/contributing/ASPNetCore-Debugging.md

```diff
@@ -1,25 +1,29 @@
 # Debugging with experimental ASP.NET Core bits
+
 Sometimes it may be necessary to make changes in [`dotnet/aspnetcore`](https://github.com/dotnet/aspnetcore), and react to the changes in this repo. The following are steps which outline the general process in using ASP.NET Core development `nupkg`s with Razor Tooling.
 
-## Steps:
+## Steps
+
 1. Checkout [`dotnet/aspnetcore`](https://github.com/dotnet/aspnetcore), and follow the initialization instructions in the [Build From Source](https://github.com/dotnet/aspnetcore/blob/main/docs/BuildFromSource.md) guide.
-2. `./restore.cmd`
-3. Make the desired changes in `dotnet/aspnetcore`.
-4. `./eng/build.cmd -pack`. The `-pack` option causes the creation of NuGet packages.
-5. You should see the generated packages in the `aspnetcore\artifacts\packages\Debug\NonShipping` directory. The packages should end with `x.0.0-dev.nupkg` where `x` is the current .NET version.
-6. Open `razor-tooling/NuGet.config` and add the local package sources:
-- `<add key="ASPNETCORE_SHIPPING" value="<PATH_TO_ASPNET_CORE_REPO>\artifacts\packages\Debug\Shipping\" />`
-- `<add key="ASPNETCORE_NONSHIPPING" value="<PATH_TO_ASPNET_CORE_REPO>\artifacts\packages\Debug\NonShipping\" />`
+1. `./restore.cmd`
+1. Make the desired changes in `dotnet/aspnetcore`.
+1. `./eng/build.cmd -pack`. The `-pack` option causes the creation of NuGet packages.
+1. You should see the generated packages in the `aspnetcore\artifacts\packages\Debug\NonShipping` directory. The packages should end with `x.0.0-dev.nupkg` where `x` is the current .NET version.
+1. Open `razor-tooling/NuGet.config` and add the local package sources:
+
+   - `<add key="ASPNETCORE_SHIPPING" value="<PATH_TO_ASPNET_CORE_REPO>\artifacts\packages\Debug\Shipping\" />`
```

Comment:
By indenting this we can avoid breaking the list into two lists. I recommend not manually numbering numbered lists since it can hide this kind of mistake.